dataactcore/migrations/versions/fc6be41471a3_populate_submission_reporting_period_.py

```diff
@@ -12,56 +12,16 @@
 branch_labels = None
 depends_on = None
 
-from alembic import op
```

Comment:
I talked about it with CM and he thinks it's better to just have it there rather than alter the history/order